declarations_site/catalog/templates/bi.jinja

```diff
@@ -1,90 +1,77 @@
-                        </div>
-                    </div>
-                </div>
-                <!-- end of content -->
+            <div role="tabpanel" class="tab-pane bi-frame" id="incomes_and_assets">
+                <iframe src='http://seim/declar/single/?appid=4086c6c3-1f0e-48ff-b100-5e5154bd89ae&sheet=1c1e23d1-d729-472f-85d3-cd477113f925&opt=nointeraction&select=clearall'></iframe>
+            </div>
+            <div role="tabpanel" class="tab-pane bi-frame" id="liabilities">
+                <iframe src='http://seim/declar/single/?appid=4086c6c3-1f0e-48ff-b100-5e5154bd89ae&sheet=bcb0b49a-4b9f-4a4f-8042-0c478d7f8e35&opt=nointeraction&select=clearall'></iframe>
+            </div>
+            <div role="tabpanel" class="tab-pane bi-frame" id="realty">
+                <iframe src='http://seim/declar/single/?appid=4086c6c3-1f0e-48ff-b100-5e5154bd89ae&sheet=17ebd8ec-ec98-4851-b1d2-a5aaebce08ad&opt=nointeraction&select=clearall'></iframe>
+            </div>
+            <div role="tabpanel" class="tab-pane bi-frame" id="vehicles">
+                <iframe src='http://seim/declar/single/?appid=4086c6c3-1f0e-48ff-b100-5e5154bd89ae&sheet=1a34f05f-8594-48eb-98df-9378a7ee5ca8&opt=nointeraction&select=clearall'></iframe>
+            </div>
+            <div role="tabpanel" class="tab-pane bi-frame" id="risks">
+                <iframe src='http://seim/declar/single/?appid=4086c6c3-1f0e-48ff-b100-5e5154bd89ae&sheet=b3d23282-6d01-4331-a6df-46cb43be7bc1&opt=nointeraction&select=clearall'></iframe>
+            </div>
+            <div role="tabpanel" class="tab-pane bi-frame" id="regions">
+                <iframe src='http://seim/declar/single/?appid=4086c6c3-1f0e-48ff-b100-5e5154bd89ae&sheet=e6c6953b-24df-41ca-a0c2-fb2076ceba12&opt=nointeraction&select=clearall'></iframe>
+            </div>
+            <div role="tabpanel" class="tab-pane bi-frame" id="list_of_declarants">
+                <iframe src='http://seim/declar/single/?appid=4086c6c3-1f0e-48ff-b100-5e5154bd89ae&sheet=0666db7e-eff0-4bc3-a05b-e073c207861c&opt=nointeraction&select=clearall'></iframe>
             </div>
         </div>
     </div>
+    <ul class="thank-to">
+        <li>
+            <a href="http://www.rbcgrp.com" target="_blank" class="rbc"></a>
+        </li>
+        <li>
+            <a href="http://en.brdo.com.ua/" target="_blank" class="brdo"></a>
+        </li>
+        <li>
+            <a href="https://bihus.info/projects/go-kancelarska-sotna" target="_blank" title="Канцелярська сотня" class="wh100">
+                <span class="svg-image inline-logo media-object"><svg preserveAspectRatio="xMidYMid" focusable="false"><use xlink:href="{{ static("images/svg-images-sprite.svg#wh100") }}"></use></svg></span>
+            </a>
+        </li>
+    </ul>
 </section>
 {% endblock %}
 
 {% block extra_js %}
-    {% javascript 'js_bi' %}
-{% endblock %}
\ No newline at end of file
+    <script type="text/javascript">
+        $('#myTabs a').click(function (e) {
```

Comment:
Move it to the external JS file, please

Reply:
we don't need this script at all, since we have data-toggle attrib at html.